package.json

```diff
@@ -40,5 +45,8 @@
             "eslint",
             "git add"
         ]
+    },
+    "dependencies": {
+        "@types/jest": "^23.3.12"
```

Comment:
This brings Jest autocompletion into VS Code.

Reply:
Oh interesting. I assume it works if this is in `devDependencies`? I feel like that's a more appropriate place since it's only used for local development purposes.